addons/hr/static/tests/m2x_avatar_employee_legacy_tests.js

```diff
@@ -0,0 +1,197 @@
+/** @odoo-module **/
+
+import {
+    start,
+    startServer,
+} from '@mail/../tests/helpers/test_utils';
+
+import { makeFakeNotificationService } from "@web/../tests/helpers/mock_services";
+
+import { Many2OneAvatarEmployee } from '@hr/js/m2x_avatar_employee';
+import { dom } from 'web.test_utils';
+import { registry } from "@web/core/registry";
+
+function clearOwlWidgets() {
+    // Remove the owl version of the widgets we want to test, must be done
+    // after the registries are set up.
+    const fieldRegistry = registry.category("fields");
+    fieldRegistry.remove("many2one_avatar_employee");
+    fieldRegistry.remove("kanban.many2one_avatar_employee");
+    fieldRegistry.remove("many2many_avatar_employee");
+    fieldRegistry.remove("kanban.many2many_avatar_employee");
+    fieldRegistry.remove("list.many2many_avatar_employee");
+}
```

Comment:
I'm a bit confused to why you don't want the field widgets. Are those the tests for the legacy fields ?

Reply:
sorry I had missed the "legacy" in the file name ...